packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -14,6 +17,13 @@ export interface GridExportOptions {
   allColumns?: boolean;
 }
 
+export interface GridCsvGetRowsToExportParams {
```

Comment:
I'm only passing the `apiRef` since most of the time users will want to use a selector, and selectors should be given the `apiRef` instead of the state whenever possible after #3663.

Not sure if we want to pass anything else.
We could pass a callback to generate them (like `getDefaultRowsToExport`), but the behavior is so simple right now that I'm not sure it would be very useful.

Anyway, I let the params as an object to be able to add things later.